### ISSUE TITLE
codeartifact login broken in v1.18.98

### ISSUE THREAD

**Issue by zoellner** [aws/aws-cli#5387]
Confirm by changing [ ] to [x] below to ensure that it's a bug:
- [x] I've gone though the [User Guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) and the [API reference](https://docs.aws.amazon.com/cli/latest/reference/)
- [x] I've searched for [previous similar issues](https://github.com/aws/aws-cli/issues) and didn't find any solution
  
**Describe the bug**
I am using the command `aws codeartifact login --tool npm --repository repo-name --domain domain-name --domain-owner 123456789012` on AWS CodeBuild after running `pip3 install awscli --upgrade --user` to login to our own npm repo. 
This was working fine in v1.18.97 but is broken in 1.18.98.

I think the relevant part in below log is "relativedelta only diffs datetime/date"

**SDK version number**
1.18.98
**Platform/OS/Hardware/Device**
What are you running the cli on?
AWS CodeBuild
**To Reproduce (observed behavior)**
Steps to reproduce the behavior
in CodeBuild run `pip3 install awscli --upgrade --user` followed by `aws codeartifact login --tool npm --repository repo-name --domain domain-name --domain-owner 123456789012`
(need to change names, accountId, and setup repos etc)

**Expected behavior**
login to codeartifact as it does in v1.18.97

**Logs/output**
Get full traceback and error logs by adding `--debug` to the command.

```
[Container] 2020/07/15 23:02:05 Running command aws codeartifact login --tool npm --repository repo-name --domain domain-name --domain-owner 123456789012

relativedelta only diffs datetime/date

[Container] 2020/07/15 23:02:07 Command did not exit successfully aws codeartifact login --tool npm --repository repo-name --domain domain-name --domain-owner 123456789012 exit status 255
[Container] 2020/07/15 23:02:07 Phase complete: INSTALL State: FAILED
[Container] 2020/07/15 23:02:07 Phase context status code: COMMAND_EXECUTION_ERROR Message: Error while executing command: aws codeartifact login --tool npm --repository repo-name --domain domain-name --domain-owner 123456789012. Reason: exit status 255
```
**Additional context**

**Comment by kyleknap** [comment]
@zoellner I can confirm the bug. I have a fix for this here: https://github.com/aws/aws-cli/pull/5382. While we work on getting this merged and in a release, I'd recommend just pinning to 1.18.97 to get your builds passing. You could also use v2 as it is unaffected by this issue.

**Comment by kdaily** [comment]
Hi @zoellner, this should be fixed in the new release `1.18.99`. Thanks again for reporting.
